docs/analyzer-rules/AZFW0008.md

```diff
@@ -8,17 +8,19 @@
 
 ## Cause
 
-This rule is triggered when the EventHubs trigger of an Azure Function is invalid.
+This rule is triggered when an Input Binding or Trigger of an Azure Function has an invalid Cardinality.
```

Comment:
nit: Noticed this throughout the code and docs, trigger is also a binding technically so this makes a little more sense to me

```suggestion
This rule is triggered when an Input or Trigger Binding of an Azure Function has an invalid Cardinality.
```

Reply:
+ the debug log!